auxiliary/auxiliary.go

```diff
@@ -13,6 +14,85 @@ import (
 	"golang.org/x/crypto/ssh/terminal"
 )
 
+// ParseArgs parses the command line arguments and returns the
+// user's configuration options for the caller to then use.
+func ParseArgs(args []string) (string, error) {
+	helpMessage := fmt.Sprintf(`Usage: %s [arguments]
+
+    help [mode] - Print detailed help regarding a mode.
+
+    add 	- Add a new secure secret to storage.
+    get 	- Retrieve a previously stored secret.
+    forget 	- Remove a previously stored secret.`, args[0])
+
+	if len(args) < 2 {
+		fmt.Println(helpMessage)
+		return "", errors.New("help")
```

Comment:
Move `errors.New()` into a package scoped variable, IE:

`var ErrYourError = errors.New("")` and then you can return `ErrYourError`, and check `if err == ErrYourError` if need be.